Update!

realyukii opened the pull request:
- add owner
- add @material-tailwind/react
- fix unexpected cannot parse json, use text instead
- and other...

realyukii committed:
Merge pull request #1 from Reyuki-san/migrate-next-13

Migrate to nextjs 13

Commit:
chore: update configuration for debugging and monitoring

Commit:
fix: unexpected cannot parse json, use text instead

Commit:
chore: add tailwind prettier plugin

Commit:
Merge branch 'migrate-next-13' of github.com:Reyuki-san/bilibili-dl into dev

Commit:
chore: add @material-tailwind/react

Commit:
chore: add @material-tailwind/react

Commit:
chore: add owner

Commit:
revert: return to pages dir

Commit:
chore: add JSDoc

Commit:
chore: add JSDoc

realyukii commented:
According to [c77a7ec](https://github.com/hansputera/bilibili-dl/pull/4/commits/c77a7ec057a97def142d49396f03c05e51d957f9)

For the sake of compatibility, I've chosen to utilize mantine and pages dir. Is that acceptable to you @hansputera @hanifdwyputras ?
At present, the app dir remains quite unstable for CSS frameworks that predominantly operate on the client side.

Commit:
feat: add layout, dark mode, path aliases

hansputera commented:
> According to [c77a7ec](https://github.com/hansputera/bilibili-dl/pull/4/commits/c77a7ec057a97def142d49396f03c05e51d957f9)
> 
> For the sake of compatibility, I've chosen to utilize mantine and pages dir. Is that acceptable to you @hansputera @hanifdwyputras ?
> At present, the app dir remains quite unstable for CSS frameworks that predominantly operate on the client side.
> 

Thanks for you contributions, I'll review it on 29 May 2023

Commit:
fix: dark mode

feat: add logo

Commit:
feat: add following component

Commit:
feat: add i18n feature, update Header component

Commit:
fix: missing jsdoc

Commit:
feat: add switcher language

fix: disable usage of cookie as workaround for IntlError: MISSING_MESSAGE: No messages were configured on the provider.

chore: add semantic naming conventions for lang format

Commit:
feat: add display suggestion hot anime with mock data

Commit:
feat: Display popular anime with Custom Card

Commit:
fix: color theme

Commit:
feat: adjusts subtitle to json format

wip: responsive layout

Commit:
feat: add swr, carousel

wip: Anime timeline component

wip: get list of popular anime

Commit:
wip: get list of popular anime

Commit:
fix: eslint prefer-spread error

Commit:
feat: delete unused endpoint

Commit:
fix: vercel error - https://github.com/vercel/next.js/discussions/49269

Commit:
fix: vercel error https://github.com/vercel/next.js/issues/49355#issuecomment-1538406708

Commit:
wip: improve some responsive layout

feat: fill Recomendation, Popular, Anime timeline component with real content, then handle it with custom hooks

chore: add general interface

Commit:
fix: eslint
